README.rst

```diff
@@ -949,6 +949,28 @@ domain, you can use this option to change the base of the url, this will
 enable you to point all the API urls to your openwisp-monitoring API server's
 domain, example: ``https://mymonitoring.myapp.com``.
 
+``OPENWISP_MONITORING_INFLUXDB_MAX_RETRIES``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    |   ``int``   |
++--------------+-------------+
+| **default**: |    ``6``    |
++--------------+-------------+
+
+This settings allow you to configure the max retries count on failure in InfluxDB operations.
```

Comment:
Let's not mention InfluxDB but use the generic "Timeseries Database".

Please also make sure these new settings are listed after `OPENWISP_MONITORING_WRITE_RETRY_OPTIONS` and make sure to explain the difference between these new settings and the old one.